### PR TITLE
[release-4.1] Bug 1732580: Best-effort loading

### PR DESCRIPTION
**Pull request by njhale**
This PR reimplements best-effort loading and default permissiveness for the `appregistry-server` command for `release-4.1`.

/hold

### CODE REVIEW

**Comment by ecordell** [comment]
/hold cancel

Couldn't reproduce the issue referenced above

File: pkg/sqlite/configmap.go [hunk 1]
```diff
@@ -100,76 +104,84 @@ func (c *ConfigMapLoader) Populate() error {
...
+		errs = append(errs, fmt.Errorf("error loading CSV list: %s", err))
+		return utilerrors.NewAggregate(errs)
 	}
 
 	var parsedCSVList []v1alpha1.ClusterServiceVersion
 	err = json.Unmarshal([]byte(csvListJson), &parsedCSVList)
 	if err != nil {
-		c.log.WithError(err).Debug("error parsing CSV list")
-		return err
+		errs = append(errs, fmt.Errorf("error parsing CSV list: %s", err))
```
Comment (ecordell) [inline pkg/sqlite/configmap.go]: This is fine given the goals of the PR are really geared towards appregistry, but -

if we wanted to be permissive here, wouldn't we treat this as a generic list, and attempt to parse each CSV one by one? As it stands now, if one doesn't parse, the whole thing will bail.

**Comment by eparis** [comment]
labeling based on request from @mfojtik

**Pull request merged**